Netinfo is always referring to RN version 0.20.1

<!-- NOTE: The iOS Simulator has some known issues with report network state changes. Please DO NOT report iOS bugs unless you are seeing them on a real device. -->

## Environment
<!-- Run `react-native info` in your terminal and paste its contents here. -->
  React Native Environment Info:
    System:
      OS: Windows 10
      CPU: (12) x64 Intel(R) Xeon(R) CPU E5-1650 v4 @ 3.60GHz
      Memory: 24.39 GB / 63.90 GB
    Binaries:
      Yarn: 1.15.2 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
      npm: 6.9.0 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: Version  3.2.0.0 AI-181.5540.7.32.5056338

## Platforms
<!-- Is this issue related to Android, iOS, or both? -->
Android

## Description
<!-- Describe your issue in detail. Include screenshots if needed. If this is a regression, let us know. -->
I am trying to create an AAR file of react-native-netinfo.
While building react-native-netinfo for android I am facing below error:
`C:\react-native-netinfo\android\src\main\java\com\reactnativecommunity\netinfo\NetInfoModule.java:14: error: package com.facebook.react.module.annotations does not exist
import com.facebook.react.module.annotations.ReactModule;
                                            ^
C:\react-native-netinfo\android\src\main\java\com\reactnativecommunity\netinfo\NetInfoModule.java:17: error: cannot find symbol
@ReactModule(name = NetInfoModule.NAME)
 ^
  symbol: class ReactModule
2 errors`

This is happening because react-native 0.20.1 is getting used from maven and in build.gradle local maven repository is not mentioned. This case has been handled in react-native-webview https://github.com/react-native-community/react-native-webview/blob/master/android/build.gradle


## Reproducible Demo
<!-- Let us know how to reproduce the issue. Include a code sample or share a project that reproduces the issue. -->
<!-- Please follow the guidelines for providing a minimal example: https://stackoverflow.com/help/mcve -->
From command line execute the following commands:
1. `git clone https://github.com/react-native-community/react-native-netinfo.git`
2. `cd react-native-netinfo`
3. `yarn`
4. `cd android`
5. `gradlew assemble`

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -62,6 +62,10 @@ android {
 }
 
 repositories {
+  maven {
+    // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
+    url "$rootDir/../node_modules/react-native/android"
+  }
   google()
   jcenter()
   mavenCentral()
@@ -69,7 +73,7 @@ repositories {
 
 dependencies {
   //noinspection GradleDynamicVersion
-  api 'com.facebook.react:react-native:+'
+  implementation 'com.facebook.react:react-native:+'
 
   def supportLibVersion = getExtOrInitialValue('supportLibVersion', getExtOrInitialValue('supportVersion', null))
   def androidXVersion = getExtOrInitialValue('androidXVersion', null)

